wdl/VcfClusterSingleChromsome.wdl

```diff
@@ -164,10 +168,37 @@ task JoinContigFromRemoteVcfs {
   
   command <<<
     set -eu -o pipefail
-    
+
+    ## This doesn't quite work (but almost!) as a replacement for tabix below
```

Comment:
It might be good to make an ticket to track these issues. Not sure if you think they should be fixed in GATK or not, but if so it might be good to make an issue there as well.

Reply:
Or if there's nothing to be fixed I might replace this chuck of commented code with a shorter comment explaining that you tried to do this with GATK and what the issues were.

Reply:
Thanks I had forgotten this was here. I've replaced it with a reference to #52.